src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -11,13 +11,14 @@
     <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.3.0</AssemblyVersion>
     <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.4.0</AssemblyVersion>
     <PackageValidationBaselineVersion>4.5.1</PackageValidationBaselineVersion>
+    <IsPlaceholderTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp2.1'">true</IsPlaceholderTargetFramework>
   </PropertyGroup>
 
-  <PropertyGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">
+  <PropertyGroup Condition="'$(IsPlaceholderTargetFramework)' == 'true'">
     <EnableDefaultItems>false</EnableDefaultItems>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' == 'true'">
     <Compile Include="TypeForwards.net.cs" />
   </ItemGroup>
```

Comment:
We shouldn't need the typeforwards file here anymore as we don't compile anything.